Sources/Player/PlaybackEngine/Internal/AudioSessionMonitors/AudioSessionInterruptionMonitor.swift

```diff
@@ -57,12 +60,15 @@
 				if let currentPlayer = playerEngine?.currentPlayer(), currentPlayer.shouldVerifyItWasPlayingBeforeInterruption {
 					if wasPlayingWhenInterrupted {
 						playerEngine?.play(timestamp: PlayerWorld.timeProvider.timestamp())
+					} else {
+						PlayerWorld.logger?.log(loggable: PlayerLoggable.interruptionMonitorHandleNotificationEndedNotPlayingWhenInterrupted)
 					}
 				} else {
 					playerEngine?.play(timestamp: PlayerWorld.timeProvider.timestamp())
 				}
 
 			default: ()
+				PlayerWorld.logger?.log(loggable: PlayerLoggable.interruptionMonitorHandleNotificationUnknownType)
```

Comment:
Not sure if `default:` is ever called here. The only two types are already handled.
I would prefer to delete the `default:` case and get a compiler error if iOS adds more types later.

Reply:
makes sense. Currently if i remove the default, compiler complains and adds `@unknown default:`.

Reply:
This log is related to my previous comment, I'm trying to tracking the flow of an issue.